docs/sources/zoom/README.md

```diff
@@ -82,6 +82,35 @@ Once the scopes are added, click on `Done` and then `Continue`.
 
 6. Activate the app
 
+## Zoom AI Metric Snapshot Bulk
+
+Psoxy can pseudonymize Zoom AI Metrics Snapshot CSV data.
+
+The default proxy rules for `zoom-ai-snapshot-bulk` will pseudonymize `User Name` and `Email`, redacting `Department`
+
+```hcl
+custom_bulk_connector_rules = {
+    "zoom-ai-snapshot-bulk" = {
+        source_kind               = "zoom-ai-snapshot",
```

Comment:
i kinda think this is `zoom` or `zoom-ai`. yeah, different connectors give us slightly different data, but it's all "zoom" data imho. Otherwise we're going to be almost 1:1 between source_kind and connector. I think source_kind is like the "tool", basically.